### PR TITLE
Update/fix documentation

### PR DESCRIPTION
**Pull request by fuentesj11**

### CODE REVIEW

File: documentation/extensions/consistent_colors.md [hunk 1]
```diff
@@ -10,14 +10,16 @@ color on all platforms to provide a consistent user experience.
 The utility class `ConsistentColor` allows the generation of colors to a given
 string following the specification of [XEP-0392](https://xmpp.org/extensions/xep-0392.html).
 
-##Usage
+**Usage**
```
Comment (vanitasvitae) [inline documentation/extensions/consistent_colors.md]: Can you change this to
```
## Usage
```
?

File: documentation/extensions/consistent_colors.md [hunk 2]
```diff
@@ -10,14 +10,16 @@ color on all platforms to provide a consistent user experience.
...
+
 To generate a consistent color for a given string, call
 ```
 float[] rgb = ConsistentColor.RGBFrom(input);
 ```
 The resulting float array contains values for RGB in the range of 0 to 1.
 
-##Color Deficiency Corrections
+**Color Deficiency Corrections**
```
Comment (vanitasvitae) [inline documentation/extensions/consistent_colors.md]: Can you change this to
```
## Color Deficiency Corrections
```
?

File: documentation/extensions/consistent_colors.md (hunk 1 above)
Comment (fuentesj11) [reply documentation/extensions/consistent_colors.md]: Sorry about that, will do.

File: documentation/extensions/consistent_colors.md (hunk 2 above)
Comment (fuentesj11) [reply documentation/extensions/consistent_colors.md]: Sorry, again.

File: documentation/extensions/consistent_colors.md (hunk 1 above)
Comment (vanitasvitae) [reply documentation/extensions/consistent_colors.md]: Don't worry :)

**Comment by Flowdalic** [comment]
The PR is way to big to review and get merged in one. Please split it into smaller commits (and PRs) so that it's easier to review. Also please don't use 'stanza(/packet)' - I know that this is how it is written in some places - but simply write 'stanza'.

**Comment by fuentesj11** [comment]
Closed to make changes as multiple pull requests

**Comment by Flowdalic** [comment]
Please also target the master branch

**Pull request closed without merging**